vsphere-RHEL-manager-blueprint-inputs.yaml

```diff
@@ -9,15 +9,15 @@
 #vsphere_port: 443
 #vsphere_datacenter_name: Datacenter
 #vsphere_resource_pool_name: Resources
-#vsphere_auto_placement: true
+#vsphere_auto_placement: false
 
 # Template
-#template: "CentOS7_1406_tmpl"
+#template: "CentOS-7-x86_64-1406"
```

Comment:
What does template mean? In general, what we want to achieve here is that each and every configuration property is documented enough so that a user will need to do as little searching as possible to answer what each property means.